deps/spidershim/include/v8.h

```diff
@@ -478,13 +480,13 @@ struct BarrierMethods {
 template <class T>
 class PersistentBase {
  public:
-  V8_INLINE void Reset();
+  void Reset();
```

Comment:
Why are you removing V8_INLINE's here?  There's some functions that are inlined at the end of this file, and these ones can be inlined there too.